posthog/warehouse/models/view_link.py

```diff
@@ -0,0 +1,13 @@
+from posthog.models.utils import UUIDModel, CreatedMetaFields, DeletedMetaFields
+from django.db import models
+from posthog.models.team import Team
+from .datawarehouse_saved_query import DataWarehouseSavedQuery
+
+
+class DataWarehouseViewLink(CreatedMetaFields, UUIDModel, DeletedMetaFields):
```

Comment:
Data model seems a bit flimsy as it's a bunch of string matching (as opposed to having an Enum of possible tables/fields)